storage/interceptors/notifications_interceptor.go

```diff
@@ -96,7 +103,7 @@ func (ni *NotificationsInterceptor) OnTxUpdate(h storage.InterceptUpdateOnTxFunc
 			}
 		}
 
-		if err := CreateNotification(ctx, repository, types.MODIFIED, updatedObject.GetType(), newPlatformID, &Payload{
+		if err := CreateNotification(ctx, repository, types.MODIFIED, updatedObject.GetType(), updatedPlatformID, &Payload{
```

Comment:
This should be in else cause - when  oldPlatformID == updatedPlatformID.

Reply:
It is true, but for brokers in k8s platforms we need to send update notification in case there are updates in the catalog in order to fetch the newly filtered catalog.
One option is to split the generic notification interceptor in two specific for visibility and brokers